Gemfile

```diff
@@ -18,11 +18,6 @@
 source "https://rubygems.org"
 gemspec
 
-# TODO when chef-dk 3.0 is released to Rubygems as 3.0 we can get rid of this
-gem "chef-dk", git: "https://github.com/chef/chef-dk.git", branch: "master"
-
-gem "train", git: "https://github.com/chef/train.git", branch: "v1.4.6"
```

Comment:
Concerned with dropping this pin - last time we tried (1.4.8?) we had health check failures in the windows build iirc.